db/migrate/20150529221342_create_items.rb

```diff
@@ -3,6 +3,7 @@ def change
     create_table :items do |t|
       t.references :user
       t.text :name
+      t.text :type
```

Comment:
Do we want type as a text attribute?

Yes. Probably. That makes the most intuitive sense. The alternative in my mind is just setting up a system where a certain byte value (say, 3) corresponds to a type. But the slight efficiency boost from switching from small text objects to bytes/ints is probably beyond negligible.